Sources/SearchableCollectionController.swift

```diff
@@ -22,33 +22,56 @@ open class SearchableCollectionController: SweetCollectionController {
     /// The installed UISearchController. Override the delegate methods (call super) if necessary.
     open lazy var searchController: UISearchController = {
         let controller = UISearchController(searchResultsController: nil)
-        controller.hidesNavigationBarDuringPresentation = true
```

Comment:
Might be good to limit this change to iOS 11

Reply:
sure, true